resource_trigger/test_sync_trigger.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for Concourse (Pipeline Trigger) Resource"""
+import check
+import pytest
+import io
+import json
+
+
+def fake_input_stream():
+    def make_stream(json_obj):
+        stream = io.StringIO()
+        json.dump(json_obj, stream)
+        stream.seek(0)
+        return stream
+
+    def make_input():
+        return {
+            "source": {
+                "archive_server": "qa.cnx.org",
```

Comment:
Might be nice to have a unit test for the archive server resolution

Reply:
What do you mean by 'archive server resolution"? do you mean a test for `determine archive server` function?

Reply:
yup!